Framework/ServiceClasses/PrinterServiceProvider/PrinterServiceClass.cs

```diff
@@ -157,6 +157,7 @@ public bool LoadSingleDefinition(string definition, bool overwrite, out XFS4IoT.
                     }
                     else
                         Medias.Add(media.Name, media);
+                    PersistentData.Store<Dictionary<string, Media>>(typeof(Media).FullName, Medias);
```

Comment:
Agreed, it's good to save field and media definition successfully loaded in the persistent file.